classes/locations.php

```diff
@@ -228,18 +231,18 @@ function geo_location( $post ) {
 			$location_lng      = get_post_meta( $post->ID, 'location_lng', true ) ?: '';
 
 			?>
-			<p class="sub"><?php _e( 'You must enter either an address or a latitude/longitude. If you enter both, the address will not be geocoded and your latitude/longitude values will remain intact.', 'SimpleMap' ); ?></p>
+			<p class="sub"><?php _e( 'You must enter either an address or a latitude/longitude. If you enter both, the address will not be geocoded and your latitude/longitude values will remain intact.', 'simplemap' ); ?></p>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 226 characters

<!-- Comment by [SideCI](https://sideci.com) -->